core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -3370,10 +3370,7 @@ class KafkaApis(val requestChannel: RequestChannel,
     } else {
       metadataSupport match {
         case ZkSupport(adminManager, controller, zkClient, forwardingManager, metadataCache, _) =>
```

Comment:
If we will eventually remove zk related code, probably we can use `case default` here and avoid to use `ZkSupport`.